.github/workflows/release.yml

```diff
@@ -1,16 +1,22 @@
 name: CI-Release
-on: 
+on:
   push:
-    branches:
-      - master
+    branches: [master, develop]
```

Comment:
This mistags releases, trunk release did not refer to the squeak version but to our trunk (develop). now master releases are also classified as trunk releases even though they are not

Reply:
Note to myself: CI is green again, but I need to test the release process manually like in aa0d83f268fcdffd0a4f094f1a9f102b38260d59 before this can be merged. I hope I'll find some time for this soon ...

Reply:
Should be implemented via abd0997? See https://github.com/LinqLover/TelegramClient/releases/tag/Trunk20201101-141029 for example.